lib/rules/no-invalid-position-declaration/__tests__/index.mjs

```diff
@@ -316,3 +317,38 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: true,
+	customSyntax: 'postcss-html',
+	accept: [
+		{
+			code: '<a style="color: red"></a>',
+			description: 'HTML style attribute',
+		},
+	],
+	reject: [
+		{
+			skip: true,
+			code: '<style>color: red</style>',
+			message: messages.rejected,
+			line: 1,
+			column: 6,
+			endLine: 1,
+			endColumn: 10,
+		},
+	],
+});
+
+testRule({
+	ruleName,
+	config: true,
+	customSyntax: naiveCssInJs,
+	accept: [
+		{
+			code: 'const foo = styled.a`color: red;`',
```

Comment:
[suggestion] This test code seems to have no sense for `naiveCssInJs`. How about fixing it like so?

https://github.com/stylelint/stylelint/blob/b17f5a421fbba4867db9667bf46934450962f639/lib/__tests__/fixtures/postcss-naive-css-in-js.cjs#L16-L17

```suggestion
			code: 'css` color: red; `;',
```